internal/rbd/group/volume_group.go

```diff
@@ -469,3 +474,296 @@ func (vg *volumeGroup) ListVolumes(ctx context.Context) ([]types.Volume, error)
 func (vg *volumeGroup) ToMirror() (types.Mirror, error) {
 	return vg, nil
 }
+
+func (vg *volumeGroup) EnableMirroring(ctx context.Context, mode librbd.ImageMirrorMode) error {
```

Comment:
I wonder if it is worth to simplify things a little by placing this in a separate file `group_mirror.go`?

```go
type volumeGroupMirror *volumeGroup // or make it a struct with extra members?

func (vgm *volumeGroupMirror) EnableMirroring(ctx context.Context, mode librbd.ImageMirrorMode) error {
    ...
```

This would keep the VolumeGroup interface smal(ler). I am considering a similar approach for VolumeGroupSnapshot.